components/scream/src/share/tests/field_tests.cpp

```diff
@@ -727,6 +729,82 @@ TEST_CASE("field_property_check", "") {
       REQUIRE(p.check(f1));
     }
   }
+
+  // Check that the values of a field are above a lower bound
+  SECTION ("field_lower_bound_check") {
+    Field<Real> f1(fid);
+    auto lower_bound_check = std::make_shared<FieldLowerBoundCheck<Real> >(-1.0);
+    REQUIRE(lower_bound_check->can_repair());
+    f1.add_property_check(lower_bound_check);
```

Comment:
Since we're just checking the new FPC, let's keep it simple: let's not store the FPC in the field, and avoid the for loop below (see next comment).

Reply:
Sounds good